README.md

```diff
@@ -112,6 +112,7 @@ Peril provides no implicit actions like that, it instead offers a JavaScript run
 - https://github.com/RxSwiftCommunity/peril
 - https://github.com/xcode-project-manager/peril-settings
 - https://github.com/loadsmart/peril-staging-settings
+- https://github.com/wearereasonablepeople
```

Comment:
Can you please link to the repo where you host your settings? The goal here is for people to find example repos

Reply:
@SD10 I updated link to the repo 😄

Reply:
💯